### PR TITLE
Load tests for public pages

### PR DESCRIPTION
**Pull request by monikasulik**
Todo:
- [x] Make sure this setup works on CircleCI
- [x] Make number of clients, hatch rate & number of requests configurable through env variables

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -9,10 +9,10 @@ requirements:
 
 FLAKE8 := flake8 .
 PYTEST := pytest . $(pytest_args)
+LOCUST := locust -f tests/locustfile.py --host=https://directory-ui-dev.herokuapp.com/ --clients=50 --hatch-rate=50 --num-reques=1000 --no-web --only-summary
```
Comment (konradko) [inline makefile]: `--host=` should use `DIRECTORY_UI_URL` env var, `clients`, `hatch-rate` and `num-reques` (is that a typo?) should use new env vars (these need to be added to `DOCKER_SET_DIRECTORY_TESTS_ENV_VARS` below and here: https://github.com/uktrade/directory-tests/blob/master/docker/directory-api/env.json

File: tests/locustfile.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    @task
+    def confirm_email(self):
+        # This checks only the case when an invalid code is given
+        self.client.get("confirm-email?confirmation_code=code")
+
+
+class RegularUser(HttpLocust):
+    task_set = PublicPages
+    min_wait = 500
```
Comment (konradko) [inline tests/locustfile.py]: env vars too

**Comment by monikasulik** [comment]
@konradko I give up. I tried everything I could think of and everything I found about using bash vars in bash commands and none of it worked :(

**Comment by monikasulik** [comment]
Right, so exceptions in the load tests still come up as green: https://circleci.com/gh/uktrade/directory-tests/13?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
But failures (as in getting back error http responses) come up red: https://circleci.com/gh/uktrade/directory-tests/14?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Is this acceptable for now? (I think for load testing this is ok, but for integration tests this would be a problem)

File: makefile (hunk 1 above)
Comment (monikasulik) [inline makefile]: Re: the url - I've set a new var as we don't actually want to use the same url for pytest and locust AFAIK.

**Comment by monikasulik** [comment]
@konradko Based on CircleCI output, I think there's some third place to which those env variables need to be added.

**Pull request merged**